Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -782,19 +792,25 @@ def create_user_iam(default_base_dn, args, mapper_out, disabled_users_group_cn):
                                             username=ad_user.get('name'),
                                             details=ad_user,
                                             action=IAMActions.CREATE_USER,
-                                            active=True)
+                                            active=False)  # the user should be activated with the IAMInitADUser script
 
             else:
+                error_code, _ = IAMErrors.BAD_REQUEST
+                error_msg = 'Failed to create user. ' + IAM_ERROR_MESSAGE
```

Comment:
You use the same message for different failures, Exceptions. Is it necessarily the same failure reason?